.github/workflows/black.yml

```diff
@@ -0,0 +1,33 @@
+# GitHub Action that uses Black to reformat the Python code in an incoming pull request.
```

Comment:
We should keep all jobs within the Concourse ecosystem. A combination of both will be confusing. Presumably we just need a python container with black installed?

Reply:
Removed as requested. Will review linting in Concourse.